include/real/integer_number.hpp

```diff
@@ -23,39 +23,39 @@ namespace boost{
 		 * of numbers.
 		 **/
 		template<typename T = int>
-		struct integer{
+		struct integer_number{
 			
 
-			T BASE = ((std::numeric_limits<T>::max()/2) +1);
+			T BASE = ((std::numeric_limits<T>::max() / 2) -1);
```

Comment:
should it be a const?

Reply:
Ok, will do that.
Should we make it static too?

Reply:
I don't think there is any benefit in making it static.

Reply:
Made it static const.